packages/office-ui-fabric-react/src/components/TextField/TextField.Props.ts

```diff
@@ -128,6 +128,12 @@ export interface ITextFieldProps extends React.HTMLProps<HTMLInputElement | HTML
    * @default false
    */
   validateOnFocusOut?: boolean;
+
+  /**
+   * Should run validation on Load?
```

Comment:
`validationDisabled` with default false would be a little more predictable (maybe a little more consistent with html props too.)

Also please fix the description here to be helpful to the customer, which goes into documentation.

Optional flag to disable input validation to execute when the value is changed.

Maybe you could be a little more specific here on exactly what happens.